src/pages/guidelines/motion/code.mdx

```diff
@@ -48,34 +48,61 @@ yarn add @carbon/motion
 
 ## Usage
 
-`@carbon/motion` supports standard, entrance, and exit easing curves in two
-motion modes: productive and expressive. You can access these curves using
-either Sass or JavaScript.
-
 ### Sass
 
-`@carbon/motion` exports a `carbon--motion` function and `carbon--motion` mixin
-that you can use to access the value of a motion curve or include that curve as
-the `transition-timing-function` for a selector. To use these helpers, you can
-do the following in your project:
+The `@carbon/type` package enables you to use typography from the IBM Design
+Language, including the type scale and fonts, along with typography design
+tokens from the Carbon Design System. It also comes with opinionated defaults
+for type styles on common elements like `h1`, `h2`, `p`, etc.
+
+You can use this package by writing the following:
 
 ```scss
-@import '@carbon/motion/scss/motion.scss';
+@use '@carbon/type';
+
+// Include type reset
+@include type.reset();
+
+// Include default type styles, targets h1, h2, h3, etc
+@include type.default-type();
+
+// Include utility classes for type-related properties
+@include type.type-classes();
 
-.my-custom-selector {
-  // Supplies the standard easing curve, using the productive mode by default
-  transition-timing-function: carbon--motion(standard);
+.selector {
+  // Include a type style
+  @include type.style('productive-heading-01');
 }
+```
+
+```scss
+@use '@carbon/motion';
+
+.selector {
+  // Set `transition-timing-function` directly
+  @include motion.motion(standard, productive);
 
-.my-custom-selector-v2 {
-  // Supplies the standard easing curve, but with the expressive mode, on the
-  // transition-timing-function property for this selector
-  @include carbon--motion(standard, expressive);
+  // Alternatively
+  transition: opacity motion.motion(standard, productive);
+
+  // Or use a duration
+  transition: opacity motion.$duration-fast-01;
 }
 ```
 
-Both the `motion` function and mixin support passing in the name of the motion
-curve and the mode you want to work in.
+#### API
```

Comment:
Formatting change: we normally add a manual line break between H4s and tables so they're not too close.

<img width="242" alt="image" src="https://user-images.githubusercontent.com/11670886/157760937-be4e9978-4004-47a6-a08b-1998b62b9bd0.png">

Reply:
let me know if this isn't what you meant

Reply:
Yes, but I think the `<br />` needs to be on its own line. Table is showing up broken now. 
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/11670886/157894014-205a557b-8a86-4a0c-8d1b-b01c398b75c5.png">